chapters/cmake_build-system_support.md

```diff
@@ -0,0 +1,92 @@
+# CMake Build System Support
+
+The Khronos OpenCL ecosystem currently dominantly uses CMake as tool of choice for build automation and unit testing. OpenCL has long long had facilities for building OpenCL applications long before the the Khronos-hosted OpenCL-SDK manifested.
```

Comment:
```suggestion
The Khronos OpenCL ecosystem currently dominantly uses CMake as tool of choice for build automation and unit testing. OpenCL has had facilities for building OpenCL applications long before the the Khronos-hosted OpenCL-SDK manifested.
```

Please check if this update makes sense - I think a few of these "longs" are unnecessary but I'm not exactly sure which ones.